### PR TITLE
3 get api categories

### PR DESCRIPTION
**Pull request by souste**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hi Stephen :) 

Please could you...
-Remove app.use(express.json()); from app.js until a POST/PATCH endpoint.
-Error handle for 404 Invalid path. i.e. /api/banana (please include a test)
-Remove Console logs from your controller.  

The rest looks good. Please send a new pull request when these are fixed. Thanks.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const { getCategories } = require("./controller/controller.js");
+const app = express();
+
+app.use(express.json());
```
Comment (hevdevs) [inline app.js:5]: remove app.use(express.json()); until we need it for a post/patch request

File: controller/controller.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const { selectCategories } = require("../model/model.js");
+
+exports.getCategories = (req, res) => {
+  selectCategories()
+    .then((categories) => {
+      console.log(categories);
```
Comment (hevdevs) [inline controller/controller.js:6]: remove console.log before merge

**Review by hevdevs — APPROVED** [review]
good to merge after removing the above!

File: app.js (hunk 1 above)
Comment (souste) [reply app.js:5]: Thanks Hev -sorry forgot to push those particular files again after Harrison's suggestions.

**Pull request merged**